ddtrace/tracer/sqlcomment.go

```diff
@@ -0,0 +1,174 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package tracer
+
+import (
+	"fmt"
+	"net/url"
+	"sort"
+	"strconv"
+	"strings"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+)
+
+// SQLCommentInjectionMode represents the mode of sql comment injection.
+type SQLCommentInjectionMode int
+
+const (
+	// CommentInjectionDisabled represents the comment injection mode where all injection is disabled.
+	CommentInjectionDisabled SQLCommentInjectionMode = 0
+	// ServiceTagsInjection represents the comment injection mode where only service tags (name, env, version) are injected.
+	ServiceTagsInjection SQLCommentInjectionMode = 1
+	// FullSQLCommentInjection represents the comment injection mode where both service tags and tracing tags. Tracing tags include span id, trace id and sampling priority.
+	FullSQLCommentInjection SQLCommentInjectionMode = 2
```

Comment:
```suggestion
	SQLInjectionDisabled SQLCommentInjectionMode = 0
	// ServiceTagsInjection represents the comment injection mode where only service tags (name, env, version) are injected.
	SQLInjectionModeService SQLCommentInjectionMode = 1
	// FullSQLCommentInjection represents the comment injection mode where both service tags and tracing tags. Tracing tags include span id, trace id and sampling priority.
	SQLInjectionModeFull SQLCommentInjectionMode = 2
```
Think `http.Status*` et al. They need to share the same prefix.